examples/nextjs-with-typescript/pages/MuxPlayer.tsx

```diff
@@ -62,7 +63,12 @@ function MuxPlayerPage() {
   const [autoplay, setAutoplay] = useState<"muted" | boolean>(INITIAL_AUTOPLAY);
   const [primaryColor, setPrimaryColor] = useState<string|undefined>(INITIAL_PRIMARY_COLOR);
   const [secondaryColor, setSecondaryColor] = useState<string|undefined>(INITIAL_SECONDARY_COLOR);
-  const [controlslist, setControlslist] = useState(INITIAL_CONTROLS_LIST);
+  const [controlsCssVars, setControlsCssVars] = useState(INITIAL_CONTROLS_CSS_VARS);
+  const [selectedCssVars, setSelectedCssVars] = useState(INITIAL_SELECTED_CSS_VARS);
+
+  useMutationObserver(mediaElRef.current?.theme.shadowRoot, useCallback(() => {
```

Comment:
question(blocking): These examples are supposed to be for reference for folks using Mux Player. Do we expect other folks will ever want this list of CSS vars? If yes, we should really expose them in a way that doesn't require drilling into the theme.

Reply:
👍 good point, does look a bit messy. I'll make it a static list.